### ISSUE TITLE
Invalid DIDL XML generated

### ISSUE THREAD

**Issue by mill1000** [gerbera/gerbera#2628]
Currently running 923479cdb59b9c2ba9f1806e26be4753e6ff498e and started receiving the following error from Hi-Fi Cast. It seems to happen when returning to the Library from the Now Playing screen.
![image](https://user-images.githubusercontent.com/6924622/167050848-ef1869b2-a114-4709-b05c-4dd80319174c.png)

I checked the logs with `-D` and I see the following DIDL being generated.
```
2022-05-05 19:06:00.751  debug: [sqlite_database.cc:237] select(): Adding select to Queue: SELECT "f"."id", "f"."ref_id", "f"."parent_id", "f"."object_type", "f"."upnp_class", "f"."dc_title", "f"."location", "f"."location_hash", "f"."auxdata", "f"."update_id", "f"."mime_type", "f"."flags", "f"."part_number", "f"."track_number", "f"."service_id", "f"."last_modified", "f"."last_updated", "rf"."upnp_class", "rf"."location", "rf"."auxdata", "rf"."mime_type", "rf"."service_id", "as"."persistent" FROM "mt_cds_object" "f" LEFT JOIN "mt_cds_object" "rf" ON "f"."ref_id" = "rf"."id" LEFT JOIN "mt_autoscan" "as" ON "as"."obj_id" = "f"."id"  WHERE "f"."location_hash" = 600438084 AND "f"."location" = 'F/mnt/seagate/media/Music/Volbeat/Rewind - Replay - Rebound/Volbeat - 04 - Last Day Under The Sun (Demo).flac' AND "f"."ref_id" IS NULL LIMIT 1
2022-05-05 19:06:00.751  debug: [thread_runner.h:134] lockGuard(): ThreadRunner: Guard for SQLiteThread - addTask SelectTask
2022-05-05 19:06:00.751  debug: [thread_runner.h:124] notify(): ThreadRunner: Notifying SQLiteThread
2022-05-05 19:06:00.751  debug: [sqlite_database.cc:466] run(): Running: SELECT "f"."id", "f"."ref_id", "f"."parent_id", "f"."object_type", "f"."upnp_class", "f"."dc_title", "f"."location", "f"."location_hash", "f"."auxdata", "f"."update_id", "f"."mime_type", "f"."flags", "f"."part_number", "f"."track_number", "f"."service_id", "f"."last_modified", "f"."last_updated", "rf"."upnp_class", "rf"."location", "rf"."auxdata", "rf"."mime_type", "rf"."service_id", "as"."persistent" FROM "mt_cds_object" "f" LEFT JOIN "mt_cds_object" "rf" ON "f"."ref_id" = "rf"."id" LEFT JOIN "mt_autoscan" "as" ON "as"."obj_id" = "f"."id"  WHERE "f"."location_hash" = 600438084 AND "f"."location" = 'F/mnt/seagate/media/Music/Volbeat/Rewind - Replay - Rebound/Volbeat - 04 - Last Day Under The Sun (Demo).flac' AND "f"."ref_id" IS NULL LIMIT 1
2022-05-05 19:06:00.751  debug: [upnp_xml.cc:232] renderObject(): Rendered DIDL: <container id="9163" parentID="14" restricted="1" childCount="11">
  <dc:title>Dr. Feelgood</dc:title>
  <upnp:class>object.container.album.musicAlbum</upnp:class>
  <dc:creator>Mötley Crüe</dc:creator>
  <upnp:albumArtist>Mötley Crüe</upnp:albumArtist>
  <upnp:artist>Mötley Crüe</upnp:artist>
  <upnp:albumArtURI>http://10.100.1.2:49152/content/media/object_id/9159/res_id/1/ext/file. Feelgood</upnp:albumArtURI>
  <dc:date>2022-05-04T21:38:23-0600</dc:date>
</container>
```

So something is obviously wrong with the albumArtURI
```
  <upnp:albumArtURI>http://10.100.1.2:49152/content/media/object_id/9159/res_id/1/ext/file. Feelgood</upnp:albumArtURI>
```

**Comment by KarlStraussberger** [comment]
Thank you for reporting.

Seems that we've got a problem with a dot in the file name. Can you share the full path of _Dr. Feelgood_.

**Comment by KarlStraussberger** [comment]
> Thank you for reporting.
> 
> Seems that we've got a problem with a dot in the file name. Can you share the full path of _Dr. Feelgood_.

Maybe just a screenshot of the detailled properties of the PC Directory entry on web UI.

**Comment by mill1000** [comment]
Here's the path
`/mnt/seagate/media/Music/Mötley Crüe/Dr. Feelgood`

I've since downgraded back to 1.10 and the issue does not occur. Later I can switch back to the latest and check those properties for you.

**Comment by KarlStraussberger** [comment]
Interesting how it comes that the folder name is used for thumbs.

**Comment by KarlStraussberger** [comment]
Fix is included in #2657
